acq4/devices/Camera/Camera.py

```diff
@@ -368,6 +372,19 @@ def objectiveChanged(self, obj=None):
             self.scopeState['objective'] = obj.name()
             self.scopeState['id'] += 1
 
+    def lightChanged(self):        
+        self.scopeState['lightSourceState'] = self.lightSource.describe()
+
+    def getLightState(self):
+        with self.lock:
```

Comment:
I'm not sure if it needs to be protected by a mutex.  I was going off of the optomech device as a template.  I'm open to whatever you think is the best way to handle this...